packages/ember-metal/lib/computed.js

```diff
@@ -117,7 +117,10 @@ function ComputedProperty(config, opts) {
       config.__ember_arity = config.length;
       this._getter = config;
       if (config.__ember_arity > 1) {
-        Ember.deprecate("Using the same function as getter and setter is deprecated");
+        Ember.deprecate(
+          "Using the same function as getter and setter is deprecated. For more information " +
+          "about this see http://emberjs.com/deprecations/v1.x/#toc_deprecate-using-the-same-function-as-getter-and-setter-in-computed-properties"
```

Comment:
You can pass the url as an option instead:
```js
Ember.deprecate(message, false, { url: 'http://path-to-deprecation' });
```

The url will be appended to the deprecation message, and will allow the Ember Inspector's deprecations tab to detect the url.